pkgs/development/python-modules/ckzg/default.nix

```
@@ -0,0 +1,47 @@
+{
+  lib,
+  fetchFromGitHub,
+  clang,
+  buildPythonPackage,
+  python,
+  pythonOlder,
+  pyyaml,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "ckzg";
+  version = "2.0.1";
+
+  pyproject = true;
+
+  disabled = pythonOlder "3.8";
+
+  src = fetchFromGitHub {
+    owner = "ethereum";
+    repo = "c-kzg-4844";
+    rev = "refs/tags/v${version}";
```

```suggestion
    tag = "v${version}";
```
New, fancy and reliable.

---

updated in https://github.com/NixOS/nixpkgs/pull/373563